docker/README.md

```diff
@@ -0,0 +1,22 @@
+# OpenSearch Benchmark Docker Image
+
+This Docker image allows users to spin up a Docker container preloaded with essential dependencies and get started with OpenSearch Benchmark faster.
+
+# Running OpenSearch Benchmark Image
+**Prerequisite:** Ensure that Docker is installed in the command line.
```

Comment:
Do you mean "via the command line"?  Perhaps "Ensure that Docker is installed."will suffice and you could provide a sample command (apt-get or yum).